cmd/pcapDumpRunner.go

```diff
@@ -0,0 +1,355 @@
+package cmd
```

Comment:
@bogdanvbalan not sure why you have two files here (`pcapDumpRunner.go` and `pcapDump.go`). Do we need to files? (not a big deal)